fluidkeys/main.go

```diff
@@ -267,6 +270,63 @@ func keyCreate() exitCode {
 	return 0
 }
 
+func keyList() exitCode {
+	fluidkeysDirectory, err := getFluidkeysDirectory()
+	if err != nil {
+		fmt.Printf("Failed to get fluidkeys directory")
+	}
+	db := database.New(fluidkeysDirectory)
+
+	keys, err := loadPgpKeys(db)
+	if err != nil {
+		panic(err)
+	}
+
+	emailAddressColumnWidth := len("Email address")
+
+	for _, key := range keys {
+		for _, id := range key.Identities {
+			emailAddressColumnWidth = Max(emailAddressColumnWidth, len(id.Name))
+		}
+	}
+
+	createdColumnWidth := 11
```

Comment:
What's the origin of this magic number, is it something like `len("31 May 2018")` ? (and if so, could it just be that instead?)